debugger_implementation.tex

```diff
@@ -298,6 +298,46 @@ \subsection{Using Program Buffer} \label{deb:mrprogbuf}
     \end{tabulary}
 \end{table}
 
+\subsection{Using Abstract Memory Access} \label{deb:mrabstract}
```

Comment:
same note: should we clarify here that this is from the perspective of the selected hart?